api/models/fixtures/multi_balancer.json

```diff
@@ -91,10 +91,28 @@
         "Ref": "WebPort80Balancer"
       }
     },
+    "WebPort80BalancerName": {
+      "Value": {
+        "Ref": "AWS::StackName"
```

Comment:
Ah I see how it's reusing {{ $balancer.LoadBalancerName }}.

I think it'd be a bit more "correct" to use Fn::GetAtt DNSName or similar to generate an output. Notice how all the other outputs are intrinsic functions (Fn::GetAtt or Ref).

But I also don't see an ELB GetAtt that just gives the name of the load balancer. Can it be inferred from the DNSName?

http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/intrinsic-function-reference-getatt.html